assets/section-footer.css

```diff
@@ -121,20 +121,15 @@
 
 .footer-block--newsletter {
```

Comment:
nothing to change here. But it's weird that we call this block like this when there isn't a `footer-block` class applied to that element in the first pace. 

And it's not actually a block but a block wrapper that can contain a newsletter block but not in all cases. 🤔 
I can create an issue for a small refactor on this.